org.aposin.licensescout.core/src/main/java/org/aposin/licensescout/exporter/TxtExporter.java

```diff
@@ -90,15 +89,16 @@ public void export(final OutputResult outputResult, final ReportConfiguration re
             final List<License> usedLicenses = getSortedLicenses(usedLicensesMap);
 
             final StringBuilder sb = new StringBuilder();
-            final String header = "License Report\n\n"
-                    + " Copyright (c) 2018 Association for the promotion of open-source insurance software \r\n"
-                    + "and for the establishment of open interface standards in the insurance industry. \r\n"
-                    + "All Rights Reserved.\r\n" + "\r\n"
-                    + "This product is licensed to you under the Apache License, Version 2.0 (the \"License\"). \r\n"
-                    + "You may not use this product except in compliance with the License. \r\n" + "\r\n"
-                    + "In the following sections you can find \r\n" + "- List of used/linked software projects\r\n"
-                    + "- Notices to the used artifacts\r\n"
-                    + "- List of used artifacts with license info and vendor plugin names\r\n" + "- License texts\n\n";
+            final String header = "License Report" + nl + nl
+                    + " Copyright (c) 2018 Association for the promotion of open-source insurance software " + nl
```

Comment:
Is it possible, as we are changing this part of the code, to use the information from the jar for the copyright information (year and vendor)? If not, please update the year to 2019 in this part of this change.

Reply:
Actually, this leads to a deeper problem. There is some sort of template for the text license report hard-coded, and it is specific to Aposin. As a general solution, the complete header (and maybe other intermediate texts) should be configurable as a template, so that text reports are of use for other parties than Aposin at all. As velocity templates are already used for generating the HTML reports, it would be an obvious choice - to not introduce another technology.

Reply:
I created a new issue for this: https://github.com/aposin/LicenseScout/issues/33

Reply:
Perfect. We will track this there.